.github/workflows/main.yml

```diff
@@ -1,40 +1,151 @@
-# This is a basic workflow to help you get started with Actions
-
 name: CI
 
-# Controls when the action will run. 
 on:
-  # Triggers the workflow on push or pull request events but only for the master branch
   push:
-    branches: [ master ]
-
-  # Allows you to run this workflow manually from the Actions tab
+    branches:
+    tags:
   workflow_dispatch:
-    inputs:
-      logLevel:
-        description: 'Log level'     
-        required: true
-        default: 'warning'
-      tags:
-        description: 'Test scenario tags'  
-
-# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+
 jobs:
-  # This workflow contains a single job called "build"
   build:
-    # The type of runner that the job will run on
-    runs-on: macos-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: ubuntu-latest
+            bazel-cache-path: ~/.cache/bazel
+            build-command: bazel build //zetasql/local_service:libremote_server.so
+            lib-path: bazel-bin/zetasql/local_service/libremote_server.so
+            artifacts-archive: libremote_server.so.tar.gz
+
+          - os: macos-latest
+            bazel-cache-path: /private/var/tmp/_bazel_*
+            build-command: |
+              bazel build //zetasql/local_service:libremote_server.dylib --features=-supports_dynamic_linker
+              install_name_tool -id "@loader_path/libremote_server.dylib" bazel-bin/zetasql/local_service/libremote_server.dylib
+            lib-path: bazel-bin/zetasql/local_service/libremote_server.dylib
+            artifacts-archive: libremote_server.dylib.tar.gz
+
+    outputs:
+      bazel-version: ${{ steps.set-bazel-version.outputs.bazel-version }}
+
+    runs-on: ${{ matrix.os }}
 
-    # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
-      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
-      - uses: actions/checkout@v2
-          
-      - name: Install Bazel
-        uses: abhinavsingh/setup-bazel@v3
+      - name: Checkout
+        uses: actions/checkout@v3
```

Comment:
Can we add dependabot for GitHub Actions dependencies? Like this https://github.com/fivetran/dbt-language-server/blob/main/.github/dependabot.yml#L8-L14.